resources/local/chainspec.toml.in

```diff
@@ -189,7 +193,11 @@ max_timestamp_leeway = '5 seconds'
 
 [transactions.v1]
 # The maximum length in bytes of runtime args per V1 transaction.
-max_args_length = 1024
```

Comment:
The limits should be presumably on the transaction size, no? Unless ModuleBytes bytes are under this also